eva_cttv_pipeline/clinvar_xml_utils.py

```diff
@@ -346,25 +346,40 @@ def nsv_id(self):
             return None
 
     @property
-    def hgvs(self):
+    def _hgvs_elems(self):
         return [
-            elem.text
+            elem
             for elem in find_elements(self.measure_xml, './AttributeSet/Attribute')
             if elem.attrib['Type'].startswith('HGVS')
         ]
 
+    @property
+    def hgvs(self):
+        return [elem.text for elem in self._hgvs_elems]
+
+
+    @property
+    def refseq_hgvs(self):
+        refseq_elems = [elem for elem in self._hgvs_elems if 'LRG' not in elem.attrib['Type']]
+        # TODO determine if there is a preferred ordering to use here, based e.g. on prevalence in data
+        return [elem.text for elem in refseq_elems]
+
     @property
     def variant_type(self):
         return self.measure_xml.attrib['Type']
 
+    @property
+    def explicit_insertion_length(self):
+        if self.vcf_alt and self.vcf_ref:
+            return len(self.vcf_alt) - len(self.vcf_ref)
+
```

Comment:
What happens if the 'if' condition is not met? What would this property return? If this is going to return None, I think it is better to do that explicitly because it's more clear.

Reply:
Agree, changed to return None explicitly.